README.md

```diff
@@ -154,7 +154,19 @@ var timeoutID = setTimeout(function[, delay]);
 
 The global `clearTimeout()` method cancels a timeout previously established by calling `setTimeout()`.
 
-see https://developer.mozilla.org/en-US/docs/Web/API/setTimeout for more information about setTimeout.
+See https://developer.mozilla.org/en-US/docs/Web/API/setTimeout for more information about setTimeout.
+
+openHAB does not return the integer timeout id as standard JS does, instead it returns an openHAB Timer.

```

Comment:
I am thinking we should have a `Timers` section with 3 subsections, so move  `SetTimeout` and `SetInterval` here and add `openHAB Timers` as well.     We then can mention in  `SetTimeout` and `SetInterval` that they return a native openHAB timer instance and to see `openHAB Timers` for more info.  In this section we can talk about rescheduling and other functions found in  https://www.openhab.org/javadoc/latest/org/openhab/core/model/script/actions/timer and give specific examples.

WDYT?

Reply:
Yeah, that should be the best option as it is well structured.
I will have a look at that.